gps_nmea_parser/src/include/gps/gps.h

```diff
@@ -145,6 +186,19 @@ typedef struct {
     uint8_t snr;                                /*!< Signal-to-noise ratio */
 } gps_sat_t;
 
+/**
+ * \brief           ENUM of possible GPS statements parsed
+ */
+typedef enum {
+    STAT_UNKNOWN    = 0,
+    STAT_GGA        = 1,
+    STAT_GSA        = 2,
+    STAT_GSV        = 3,
+    STAT_RMC        = 4,
+    STAT_UBX        = 5,
+    STAT_UBX_TIME   = 6,
+    STAT_CHECKSUM_FAIL = UINT8_MAX
+}__attribute__((packed)) gps_statement_t;
```

Comment:
This is super wrong and is not standard C. Remove this attribute.

Reply:
ok, see 15436ea9